data/oba_from_uniswap/instances/WETH-WETH.json

```diff
@@ -0,0 +1,9097 @@
+{
+  "orders": [
+    {
+      "sellToken": "WETH",
```

Comment:
Weth -weth pair does not make sense :)

Reply:
This is odd! If you look at the convoluted paths, it seems like someone is doing arbitrage :)